core/services/vrf/listener_v2.go

```diff
@@ -635,7 +652,7 @@ func (lsn *listenerV2) processRequestsPerSub(
 	start := time.Now()
 	var processed = make(map[string]struct{})
 	startBalanceNoReserveLink, err := MaybeSubtractReservedLink(
-		lsn.l, lsn.q, startBalance, lsn.ethClient.ChainID().Uint64(), subID)
+		lsn.q, startBalance, lsn.ethClient.ChainID().Uint64(), subID)
 	if err != nil {
 		lsn.l.Errorw("Couldn't get reserved LINK for subscription", "sub", reqs[0].req.SubId)
```

Comment:
Similarly here, maybe add the `err` to the log fields.

Reply:
Done